File: parOImpar/ParOImpar.f90
program ParOImpar
    implicit none
    integer :: numero

    print *, 'Ingresa un numero: '
    read *, numero

    if (mod(numero, 2) == 0) then
        print *, 'Par'
    else
        print *, 'Impar'
    end if

    print *, 'Presiona ENTER para salir...'
    read *
end program ParOImpar

File: parOImpar/ParOImpar.java
package parOImpar;

import java.util.Scanner;
/**
 *
 * @author huijo
 */
public class ParOImpar {
    
    public static void main(String[] args) {
       
        Scanner sc = new Scanner(System.in);
        
        System.out.println("Ingresa un numero: ");
        
        int numero = sc.nextInt();
        
        if (numero % 2 == 0){
            
            System.out.println("Par");
        }else {
            System.out.println("Impar");
        }
        sc.close();
    }
    
}


